package.json

```diff
@@ -18,7 +18,8 @@
     "url": "git@github.com:runspired/ember-gestures.git"
   },
   "engines": {
-    "node": ">= 0.12.0"
+    "node": "4.x.x",
+    "npm": "2.x.x"
```

Comment:
debatable but found npm > 2 to cause problems and 5.x to work, but only with npm 2, which is not its default